tests/schedule.go

```diff
@@ -1115,6 +1115,111 @@ func (s *ScheduleFunctionalSuite) TestRateLimit() {
 	s.testCluster.host.workerService.RefreshPerNSWorkerManager()
 }
 
+func (s *ScheduleFunctionalSuite) TestListSchedulesReturnsWorkflowStatus() {
+	// TODO - remove when ActionResultIncludesStatus becomes the active version
+	prevTweakables := scheduler.CurrentTweakablePolicies
+	scheduler.CurrentTweakablePolicies.Version = scheduler.ActionResultIncludesStatus
+	defer func() { scheduler.CurrentTweakablePolicies = prevTweakables }()
+
+	s.testCluster.host.workerService.RefreshPerNSWorkerManager()
```

Comment:
I don't think we need this, do we?

Reply:
You're right, we don't need the `Refresh`. Will fix.